source/preview/partials/footer.hbs

```diff
@@ -0,0 +1,41 @@
+<footer class="sg__footer" role="contentinfo">
```

Comment:
To me this is also part of it: It gives a way more professional view to the product, and it serves to me as default only. 
Proposal: I'll add some HBS vars so that it gets automatically populated wit project info. Fine for you?